modules/dev-tools/config/webpack.config.js

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017 Uber Technologies, Inc.
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
we'll have to run the script again once we've finished the current effort. I suspect not every file has the copyright notice anyway.

Reply:
Is there an official script to add these headers?